mllib/src/main/scala/org/apache/spark/ml/r/LDAWrapper.scala

```diff
@@ -172,6 +187,8 @@ private[r] object LDAWrapper extends MLReadable[LDAWrapper] {
       model,
       ldaModel.logLikelihood(preprocessedData),
       ldaModel.logPerplexity(preprocessedData),
+      trainingLogLikelihood,
+      logPrior,
```

Comment:
With the same dataset, Scala side tests:
Original LogPrior:-3.3387459952856338
LogPrior from saved model: -0.9202435107654922